.github/workflows/pr.yaml

```diff
@@ -7,6 +7,7 @@ on:
     paths:
       - '**.lock'
       - 'package-lock.json'
+      - 'report-new.sarif'
```

Comment:
@DmitriyLewen could you tell me why you did it? thanks

Reply:
i didn't change yarn lock file in this PR.
Without this change `PR action` didn't run.